src/more-infos/more-info-cover.html

```diff
@@ -36,7 +35,7 @@
           on-change='coverPositionSliderChanged'></paper-slider>
       </div>
 
-      <div class='current_tilt_position'>
+      <div class='current_tilt_position' hidden$="[[!supportsTilt]]">
```

Comment:
* What will be height of this element?
* When hidden it will still take space.

I propose you just change `has-current_tilt_position` to depend on `supportsTilt`